geth/jail/jail_cell.go

```diff
@@ -1,74 +1,50 @@
 package jail
```

Comment:
I guess it was missed during refactoring, but as we changed `JailCell` to `Cell`, maybe we can also change this file name to `cell.go`?

Reply:
@adambabik oh right, I was thinking about the same. Thanks, will do now!

Reply:
Nope. Please, add to a refactoring issue. No renames during regular issues.